content/en/blog/_posts/2024-05-13-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: blog
+title: "Gateway API v1.1: Feature Graduation and Other Updates"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+author: >
+  Richard Belleville (Google),
+  Frank Budinsky (IBM),
+  Arko Dasgupta (Tetrate),
+  Flynn (Buoyant),
+  Candace Holman (Red Hat),
+  John Howard (Solo.io),
+  Mattia Lavacca (Kong),
+  Christine Kim (Isovalent),
+  Grant Spence (Red Hat),
+  Gina Yeh (Google),
+  and other reviewer and release note contributors.
+---
+
+![Gateway API Logo](gateway-api-logo.svg)
+
+In the wake of the GA release of Gateway API last October, the Kubernetes
+SIG Network buoyantly announces the v1.1 release of
+[Gateway API](https://gateway-api.sigs.k8s.io/). Gateway API features for
+**Gateway API for Service Mesh (GAMMA)**, **GRPCRoute**, **ParentReference Port**,
+and **Conformance Profiles and Reports** will graduate to Standard.
+In addition, **Gateway Client Certificate Verification** and **Session
+Persistence** are adopted as new, Experimental features. Clarifications to TLS
+terminology and gateway merging are added to strengthen the release.
+
+## What's new
+
+### Graduation to Standard
+This release includes the graduation to _Standard_ of four eagerly awaited features.
+This means they are no longer experimental concepts. Inclusion in the Standard
+release channel denotes a high
+level of confidence in the API surface and provides guarantees of backward
+compatibility.  Though these features are now considered stable, that does not
+mean that they are complete. These and other features will continue to evolve via
+the Experimental channel. For more information on how all of this works, refer
```

Comment:
Not quite sure how to say this, but I'm worried that if we mention experimental channel here we may give the notion that unstable changes may happen to these features or that new features will only ever be available in experimental channel. Trying to avoid either of those potential points of confusion, but open to alternatives here.
```suggestion
mean that they are complete. In the future, backwards compatible additions may be made
such as new features or supported values. For more information on how all of this works, refer
```

Reply:
Maybe

> This release includes the graduation to Standard of four eagerly awaited features. This means they are no longer experimental concepts; inclusion in the Standard release channel denotes a high level of confidence in the API surface and provides guarantees of backward compatibility. Of course, as with any other Kubernetes API, Standard channel features can continue to evolve with backward-compatible additions over time, and we certainly expect further refinements and improvements to these new features in the future. For more information on how all of this works, refer to the [Gateway API Versioning Policy](https://gateway-api.sigs.k8s.io/concepts/versioning/).

?

Reply:
That's great @kflynn, thanks! Let's go with that